pkg/querier/queryrange/query_range_test.go

```diff
@@ -113,11 +113,15 @@ func TestMergeAPIResponses(t *testing.T) {
 		input    []Response
 		expected Response
 	}{
-		// No responses shouldn't panic.
+		// No responses shouldn't panic and return a non-empty result and result type.
```

Comment:
This comment is bit misleading ... it returns empty (but not null) result now.

Reply:
Is this better?